### DISCUSSION TITLE
Newbie with evertything here

### DISCUSSION THREAD

**Discussion by bermudaboy** [vogler/free-games-claimer#247] [Q&A]
i was just wondering if it was possible to have it where i can put my emails in the config and have it auto do everything, i also am unsure to to create the task within scheduler, like do i have it run docker... cmd... i am unsure

apologies about the questions.

im wiilling to do what i am told to do, i just have no idea what to do atm

**Comment by vogler** [comment]
Just run the docker command. It will ask you to login and should afterwards stay logged in.
Once that works, you can see https://github.com/vogler/free-games-claimer?tab=readme-ov-file#docker on how to set options.

> i just have no idea what to do atm

Yea, the readme is probably too confusing for non-developers; should have some gif/video at the top that shows an example.

**Reply by bermudaboy** [reply]
If possible, could a video be made, that way minimal mistakes will happen for new people in the future?

When I run it, am I meant to answer with cmd?.... Docker? 

I apologise for such ignorance

**Reply by vogler** [reply]
You can run it with cmd and later paste it to a .bat file which you can then put in autostart or start via the task scheduler.
